site/content/en/docs/start/_index.md

```diff
@@ -176,6 +176,9 @@ It may take a moment, but your deployment will soon show up when you run:
 kubectl get services hello-minikube
 ```
 
+> :warning: On M1-based machines (`arm64` architecture) the `echoserver:1.4` image currently fails to start.
```

Comment:
That  `:warning:` doesn't really work here
![warning](https://user-images.githubusercontent.com/10364051/115137934-495f0780-a029-11eb-8bab-43914428a190.png)

https://deploy-preview-11125--kubernetes-sigs-minikube.netlify.app/docs/start/

Reply:
Ok, I will correct that.

Reply:
I think using bold text would help draw this to the users eyes